mfttool/mfttool.c

```diff
@@ -136,9 +137,11 @@ int main(int argc, char *argv[])
         }
         if (r_name == NULL)
             errx(1, ".devices[...]: missing .name");
-        /*
-         * TODO: Validate that name is [A-Z][a-z][0-9]+.
-         */
+        if (strlen(r_name) > MFT_NAME_MAX)
+            errx(1, ".devices[...]: name too long");
```

Comment:
would it make sense to verify that the `r_name[0] != 0`, ie that it is not empty?

Reply:
Done